### DISCUSSION TITLE
关于vue cesium执行效率的问题？

### DISCUSSION THREAD

**Discussion by cxq201708** [zouyaoji/vue-cesium#258] [Q&A]
如题，vue data中的数据会进行数据劫持？那载数据量大的数据集，如几万个点或者3DTiles，效率如何？

**Comment by zouyaoji** [comment]
具体没测试过哈，不同配置不同数据都有差异。
不把cesium的对象存到data中还是问题不大的哦。

**Reply by Eva-Skye** [reply]
vue3也会有这种问题么？
